manifests/backup/client.pp

```diff
@@ -1,14 +1,43 @@
 class profiles::backup::client (
-  String $private_key,
-  Hash   $configuration = {}
-) inherits ::profiles {
+   String $private_key = lookup('data::backup::client::private::key', Optional[String], 'first', undef),
+   Hash   $configurations,
+ ) inherits ::profiles {
 
   realize Apt::Source['publiq-tools']
 
   Sshkey <<| title == 'backup' |>>
 
+  $borg_user      = 'borgbackup'
+  $borg_server    = 'backup.services.publiq.be'
```

Comment:
Deze zou ik een parameter maken met een hiera lookup zoals voor de private key (bv. data::backup::server). Dan kunnen we het overriden voor specifieke clients als dat ooit nodig is